### PR TITLE
fix: unable to launch VSCode in web version

### PR DESCRIPTION
**Pull request by adrysn**
### Fixes
- Unable to open the VSCode app in the web version.
  + Previous implementation always opens the VSCode Remote Connection dialog when the VSCode icon is clicked, even in the web version.
  + Separate `vscode-desktop` app from the legacy `vscode` app. The legacy code-server application is needed even in the Web UI desktop app. We cannot assume every user installs the Visual Studio Code application on their local machine. Even worse, in some cases, the users may not be allowed to install any application. So, in the desktop Web UI application, there will be two icons, one for the code-server and the other for the local VSCode.
    image
- Do not expose the VSCode Desktop icon for the core version less than 23.03.
  + The core's [counterpart PR](https://github.com/lablup/backend.ai/pull/751) is for 23.03, but not backported to 22.09.
- Remove the annoying error message, unable to create a compute session, which is always displayed when the VSCode Remote Connection dialog opens up.

### Refactors
- Update dialog messages to help users understand better.
- Adjust dialog elements' margins to reduce the total height.
image
image

Follow up of https://github.com/lablup/backend.ai-webui/pull/1487

### CODE REVIEW

File: src/components/backend-ai-app-launcher.ts [hunk 1]
```diff
@@ -499,13 +499,19 @@ export default class BackendAiAppLauncher extends BackendAIPage {
...
       }
     });
-    if (globalThis.isElectron && !appServices.includes('vscode')) {
-      this.appSupportList.push({ // Force push vscode
-        'name': 'vscode',
-        'title': 'VS Code',
-        'category': '2.Development',
-        'redirect': '',
-        'src': './resources/icons/vscode.svg'
+
```
Comment (lizable) [inline src/components/backend-ai-app-launcher.ts:503]: Somewhat I cannot see two version of vscode (vscode execution in local, vscode with web application.) in desktop app version under the condition; the latest commit of this PR branch.
I tested with [`23.03.0`](https://github.com/lablup/backend.ai/releases/tag/23.03.0) version of Backend.AI Core.

Comment (lizable) [reply src/components/backend-ai-app-launcher.ts:503]: Screenshot 2023-04-10 at 3 25 28 PM

**Review by lizable — CHANGES_REQUESTED** [review]
@adrysn 
I cannot see vscode-desktop application icon in app-launcher dialog.
Is there any special condition to detect it except for version? I already checked it in Backend.AI Core 23.03

File: src/lib/backend.ai-client-esm.ts [hunk 2]
```diff
@@ -594,6 +594,7 @@ class Client {
     }
     if (this.isManagerVersionCompatibleWith('23.03')) {
       this._features['inference-workload'] = true;
+      this._features['local-vscode-remote-conneciton'] = true;
```
Comment (lizable) [inline src/lib/backend.ai-client-esm.ts]: I found minor typo. Let's change it to `local-vscode-remote-connection`.

**Review by lizable — APPROVED** [review]
LGTM.
It seems working fine in 23.03 and 22.09.

**Review by 1HAH — APPROVED** [review]

**Review by studioego — APPROVED** [review]

**Pull request merged**